spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -419,6 +419,12 @@ public class RolesEndpoint {
 
 ```
 
+If there is no JWT in the request, an empty `JsonWebToken` is injected.  Note that MP JWT will still perform
+Authentication and Authorization if the endpoint requires these verifications. Effectively, the injected empty token
+is only visible on unauthenticated and unauthorized endpoints.
```

Comment:
+1 - this is very precise

Reply:
@radcortez Perhaps we need to add something here which would clarify why we even try to support it. For example: 
```
Particularly, the endpoints which need to control the authentication process themselves or only need to propagate the tokens further can check if a token is available by calling a `JsonWebToken.getRawToken()` method.
```
Something like that I guess.

Reply:
@radcortez I believe we need to add, immediately after Effectively, the injected empty token is only visible on unauthenticated and unauthorized endpoints. that
Token verification must still be performed if the token is available as I don't think we want to allow supporting the cases where anyone can drop whatever the want into JsonWebToken :-), of course we can't control anything at the MP JWT level spec, but if the users just want a raw token they they can just get Authorization header, but whatever goes into JsonWebToken must be verified.
The issue of not even validating the tokens for the public endpoints is really container specific. We can relax this requirement if there will be some pushback, but in 1.2 I believe we should start from a more conservative approach